cpp/benchmarks/copying/contiguous_split.cu

```diff
@@ -81,20 +74,17 @@ void BM_contiguous_split(benchmark::State& state)
   int64_t const num_rows  = total_desired_bytes / (num_cols * el_size);
 
   // generate input table
-  srand(31337);
-  auto valids = cudf::detail::make_counting_transform_iterator(0, [](auto i) { return true; });
-  std::vector<cudf::test::fixed_width_column_wrapper<int>> src_cols(num_cols);
-  for (int idx = 0; idx < num_cols; idx++) {
-    auto rand_elements =
-      cudf::detail::make_counting_transform_iterator(0, [](int i) { return rand(); });
-    if (include_validity) {
-      src_cols[idx] = cudf::test::fixed_width_column_wrapper<int>(
-        rand_elements, rand_elements + num_rows, valids);
-    } else {
-      src_cols[idx] =
-        cudf::test::fixed_width_column_wrapper<int>(rand_elements, rand_elements + num_rows);
-    }
-  }
+  data_profile profile;
+  if (not include_validity) profile.set_null_frequency(-1);  // <0 means, no null_mask
```

Comment:
Is it feasible to also change this function to use optionals the way null mask creation now does? Feel free to say no if you think it requires too many changes (I assume this value eventually gets passed to calling that function in some way).

Reply:
replaced with optional.